vault-plugins/flant_iam/model/role.go

```diff
@@ -161,3 +180,72 @@ func (r *RoleRepository) Sync(objID string, data []byte) error {
 
 	return r.save(role)
 }
+
+type includedRolesIndexer struct{}
+
+func (_ includedRolesIndexer) FromArgs(args ...interface{}) ([]byte, error) {
+	if len(args) != 1 {
+		return nil, ErrNeedSingleArgument
```

Comment:
I can't see where we process this error. I usually export errors if only they are supposed to be processed somewhere. That's why I declare them in advance.

Additionally, this error is exported. All exported errors are parts of the API of its module. So I expect they are used somewhere outside the module. The visibility creates assumptions. 

Since this method (FromArgs) is also exported, I expect the error will also be used somewhere outside. LGTM then.

Reply:
just to decrease unuseful computations